python/sglang/srt/server_args.py

```diff
@@ -482,6 +496,24 @@ def check_server_args(self):
             self.disable_flashinfer = False
 
 
+def prepare_server_args(args: argparse.Namespace) -> ServerArgs:
+    """
+    Prepare the server arguments from the command line arguments.
+
+    Args:
+        args: The command line arguments. Typically, it should be `sys.argv[1:]`
```

Comment:
The default value of `parse_args` when no arguments are passed to the function is `sys.argv[1:]`.

https://github.com/python/cpython/blob/c3ed775899eedd47d37f8f1840345b108920e400/Lib/argparse.py#L1865